### PR TITLE
reduce querycoord unnecessary panic

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: Wei Liu <wei.liu@zilliz.com>

/kind improvement

### CODE REVIEW

File: internal/querycoordv2/task/task.go [hunk 1]
```diff
@@ -44,6 +45,12 @@ const (
 	TaskPriorityHigh
 )
 
+var (
+	ErrEmptyActions          = errors.New("actions could not be empty")
+	ErrActionsTypeMismatch   = errors.New("actions have different type")
+	ErrActionsTargetMismatch = errors.New("actions have diffrerent target channel/segment")
```
Comment (congqixia) [inline internal/querycoordv2/task/task.go:51]: NIT: typo

File: internal/querycoordv2/task/task_test.go [hunk 2]
```diff
@@ -867,54 +879,96 @@ func (suite *TaskSuite) TestChannelTaskReplace() {
...
+		err = suite.scheduler.Add(task)
 		suite.NoError(err)
 	}
 	channelNum := len(suite.subChannels)
 	suite.AssertTaskNum(0, channelNum, channelNum, 0)
 }
 
+func (suite *TaskSuite) TestCreateTaskBehavior() {
+	chanelTask, err := NewChannelTask(context.TODO(), 5*time.Second, 0, 0, 0)
+	suite.Error(err)
```
Comment (yah01) [inline internal/querycoordv2/task/task_test.go:901]: redundant check

**Pull request merged**